chapter_recommender-systems/autorec_vn.md

```diff
@@ -127,7 +132,12 @@ Dropout is included after the encoding transformation to reduce over-fitting.
 The gradients of unobserved inputs are masked out to ensure that only observed ratings contribute to the model learning process.
 -->
 
-*dịch đoạn phía trên*
+Một bộ tự mã hoá điển hình bao gồm một bộ mã hoá và một bộ giải mã.
+Bộ mã hoá chiếu đầu vào thành dạng biểu diễn ẩn và bộ giải mã ánh xạ tầng ẩn tới tầng khôi phục.
+Ta tuân theo cấu trúc này và tạo bộ mã hoá và bộ giải mã với các tầng kết nối dày đặc.
+Hàm kích hoạt của bộ mã hoá được đặt mặc định bằng `sigmoid` và ta sẽ không áp dụng hàm kích hoạt nào lên tầng giải mã.
+Dropout được thêm vào sau biến đổi mã hoá nhằm giảm hiện tượng quá khớp.
+Gradient của các đầu vào chưa biết được lọc ra để đảm bảo rằng chỉ có các đánh giá đã biết tham gia vào quá trình học của mô hình.
```

Comment:
```suggestion
Gradient của các đầu vào chưa biết được che mặt nạ để đảm bảo rằng chỉ có các đánh giá đã biết tham gia vào quá trình học của mô hình.
```

Reply:
`mask out` theo em tra được thì nghĩa là `giấu` hoặc `che đi`.
Ở đây câu này theo em hiểu nghĩa là `Những tương tác chưa biết được giấu đi (loại bỏ), chỉ xét đến những tương tác đã biết` nên em dùng từ `lọc ra`.
`che mặt nạ` thì có vẻ không hợp cho lắm.

Reply:
Mình thấy dùng từ "che mặt nạ" nghe hợp lý và sát nghĩa. Trong ngôn ngữ lập trình cũng có thuật ngữ "bit masking" = mặt nạ bit (https://qastack.vn/programming/10493411/what-is-bit-masking).

Reply:
Thảo luận từ [trang qastack](https://qastack.vn/programming/10493411/what-is-bit-masking) là dịch từ thảo luận tiếng Anh trên StackOverflow mà anh.
Thứ hai là theo em tìm kiếm, từ `mặt nạ` thường được dùng dưới dạng danh từ trong cụm từ `mặt nạ bit` hay `hàm mặt nạ`. Còn `che mặt nạ` em chỉ thấy xuất hiện trong [trang dịch Tiếng Việt này của Google](https://support.google.com/analytics/answer/2763052?hl=vi) dưới dạng `che mặt nạ IP`, tức cũng không phải một tài liệu dịch chuẩn.
Em thấy khi người mới học đọc từ `che mặt nạ` sẽ dễ bị hiểu nhầm. Em nghĩ nếu cần thì có thể tạo thảo luận thêm.

Reply:
Em thử tạo thảo luận xem có ai có ý kiến gì khác không. Nếu có cách dịch nào đó mà giữ được chữ "mặt nạ" thì tốt.

Reply:
#2811 nhé anh

Reply:
```suggestion
Gradient của các đầu vào chưa biết bị che lại để đảm bảo rằng chỉ có các đánh giá đã biết tham gia vào quá trình học của mô hình.
```